cli/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@cube-creator/cli",
+  "version": "0.0.0",
+  "description": "Runs a pipeline to create an RDF data cube",
+  "main": "index.js",
+  "scripts": {
+    "postinstall": "mkdir -p output",
+    "transform": "dotenv -e ./.env -e ./.test.env -- yarn start",
+    "start": "ts-node index.ts transform --project \"$TRANSFORM_PROJECT\" --to stdout --debug --enable-buffer-monitor",
```

Comment:
The run command is simplified where everything is by default provided as environment variables